src/pages/sessions/[id]/controls.tsx

```diff
@@ -72,8 +72,10 @@ const SessionControlsPage: NextPage = () => {
                 >
                   <p className="flex items-center space-x-2">
                     <span>
-                      {item.user.name}{' '}
-                      {!usersIdsInSession?.includes(item.user.id) && (
+                      {item.user.name}
+                      
+                      {!usersInSession?.includes(item.user.id) && 
```

Comment:
`usersInSession` is a variable that doesn't exist, which leads to compilation errors. I dont think you need to change this file.

Reply:
So I should remove that line? it must have been added during rebase

Reply:
No this should not be here. Master branch isn't like this. Maybe you rebased an older version. Otherwise this diff wouldn't look like this.